gms/feature_service.cc

```diff
@@ -230,6 +231,9 @@ void feature::enable() {
 }
 
 db::schema_features feature_service::cluster_schema_features() const {
+    if (!get_local_gossiper().is_enabled()) {
+        return db::schema_features::full();
+    }
```

Comment:
Seems very dangerous. Features should be monotonicically increasing with time. With the change, they can decrease.

The idea around features is that you don't use something before you are sure everyone understands it. If you remove features, you violated this principle.

Reply:
Fair point. I guess I could just make sure that the codepath run before gossip is started to not remove anything from the schema at all when it loads tables schemas from disk.